<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_func</class>
 <widget class="QWidget" name="Add_func">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>536</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>536</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加联系人</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加联系人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="photo_pb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>添加图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_lable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>姓名      </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>164</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>电话号码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phone_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>生日</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>emai     </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>备注    :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="specialinfo_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>162</y>
     <width>106</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>分组：  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>同学</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>同事</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>亲戚</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>朋友</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="newtype_lb">
      <property name="text">
       <string>分组名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newtype_le"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
